sig-network/charter.md

```diff
@@ -0,0 +1,45 @@
+# SIG Network
+
+## Scope
+
+SIG Network is owning everything related to KubeVirt networking.
+The primary objective of the SIG is to connect the Virtual Machine to the pod network/s,
+allowing migration and other advantages of virtualization in addition to maintaining services,
+port-forwarding, service meshes, and other Kubernetes capabilities.
+
+### In scope
+Some notable examples of features owned by SIG Network:
+- Masquerade, Bridge and Passt binding plugins for the primary network.
+- Bridge, SRIOV and MacVTap bindings for secondary networks.
+- VM networking lifecycle on: creation, migration, hotplug/unplug, restart, stop etc.
+
+#### Code, Binaries and Services
+- Networking related APIs on: VM, VMI, Instance types, Kubevirt CR, etc.
+- Core network binding infrastructure.
+- Network binding plugin infrastructure.
+- Examples of binding plugin CNIs and sidecars.
+- Virtual Machine/Virtual Machine Instance networking status management.
+- Domain networking configuration.
+- Unit and e2e tests.
+
```

Comment:
What about:
- Developer and user documentation around networking
- What about help for onboarding new sig network members?
- What about grooming network related bugs?

Reply:
Added. But I"m not sure how the last two fit the `Code, Binaries and Services`. Those are services?